docs/releases/4.0.txt

```diff
@@ -37,6 +37,10 @@ Minor features
 * The ``admin/base.html`` template now has a new block ``header`` which
   contains the admin site header.
 
+* The new :meth:`ModelAdmin.get_formset_kwargs()
+  <django.contrib.admin.ModelAdmin.get_formset_kwargs>` method provides
+  the initial parameters to admin inline formsets.
```

Comment:
Please rephrase this. It doesn't expressly provide "initial parameters" - yes, you could use it to do this - but allows overriding the keyword arguments passed to `FormSet`.